### PR TITLE
translate concepts/Garbage Collection in Chinese

### LINKED ISSUE #29837: [zh] translate concepts/Garbage Collection
**This is a Feature Request**

translate /en/docs/concepts/architecture/garbage-collection.md  into Chinese.

**What would you like to be added**

/zh/docs/concepts/architecture/garbage-collection.md

/language zh
/assign

### PR DESCRIPTION
**Pull request by xzyangNB**
fix #29837

### CODE REVIEW

File: content/zh/docs/concepts/architecture/garbage-collection.md [hunk 1]
```diff
@@ -0,0 +1,181 @@
+---
+title: Garbage Collection
```
Comment (sftim) [inline content/zh/docs/concepts/architecture/garbage-collection.md:2]: I'm not sure but I think this title  should be localized

File: content/zh/docs/concepts/architecture/garbage-collection.md [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+
+<!-- overview -->
+{{<glossary_definition term_id="garbage-collection" length="short">}} 这
+允许像下面这样清理资源:
+
+  * [Failed pods](/docs/concepts/workloads/pods/pod-lifecycle/#pod-garbage-collection)
+  * [Completed Jobs](/docs/concepts/workloads/controllers/ttlafterfinished/)
+  * [Objects without owner references](#拥有者)
+  * [Unused containers and container images](#容器镜像)
+  * [Dynamically provisioned PersistentVolumes with a StorageClass reclaim policy of Delete](/docs/concepts/storage/persistent-volumes/#delete)
```
Comment (sftim) [inline content/zh/docs/concepts/architecture/garbage-collection.md:15]: Where Chinese pages exist, eg https://kubernetes.io/zh/docs/concepts/workloads/pods/pod-lifecycle/, I would link to those.

File: content/zh/docs/concepts/architecture/garbage-collection.md [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+  * [配置Kubernetes对象的级联删除](/docs/tasks/administer-cluster/use-cascading-deletion/)
+  * [配置已完成作业的清理](/docs/concepts/workloads/controllers/ttlafterfinished/)
+  
+<!-- * [配置未使用的容器和映像垃圾回收 ](/docs/tasks/administer-cluster/reconfigure-kubelet/) -->
+
+## {{% heading "whatsnext" %}}
+
+* 了解更多关于 [ownership of Kubernetes objects](/docs/concepts/overview/working-with-objects/owners-dependents/).
+* 了解更多关于 Kubernetes [finalizers](/docs/concepts/overview/working-with-objects/finalizers/).
+* 了解更多关于 [TTL controller](/docs/concepts/workloads/controllers/ttlafterfinished/) (beta) that cleans up finished Jobs.
```
Comment (sftim) [inline content/zh/docs/concepts/architecture/garbage-collection.md:181]: I'd link to localized pages, eg, https://kubernetes.io/zh/docs/concepts/overview/working-with-objects/owners-dependents/

Also “that cleans up finished Jobs” should be localized, even if there's no Chinese version of https://k8s.io/docs/concepts/workloads/controllers/ttlafterfinished/ available

**Review by sftim — CHANGES_REQUESTED** [review]
I'm afraid I think this page needs more work.

**Review by Arhell — COMMENTED** [review]
/assign @tengqm

File: content/zh/docs/concepts/architecture/garbage-collection.md (hunk 1 above)
Comment (xzyangNB) [reply content/zh/docs/concepts/architecture/garbage-collection.md]: > 

I have modified it

File: content/zh/docs/concepts/architecture/garbage-collection.md (hunk 2 above)
Comment (xzyangNB) [reply content/zh/docs/concepts/architecture/garbage-collection.md]: > 

I have modified it

File: content/zh/docs/concepts/architecture/garbage-collection.md (hunk 3 above)
Comment (xzyangNB) [reply content/zh/docs/concepts/architecture/garbage-collection.md]: > 

I have modified it

**Comment by xzyangNB** [comment]
> I'm afraid I think this page needs more work.
@sftim This is my first pr. I found that some checks failed. Will this have an impact?

**Comment by xzyangNB** [comment]
> 恐怕我认为这个页面需要更多的工作。

I have modified it

File: content/zh/docs/concepts/architecture/garbage-collection.md [hunk 4]
```diff
@@ -0,0 +1,181 @@
+---
+title: 垃圾收集
+content_type: concept
+weight: 50
+---
+
+<!-- overview -->
+{{<glossary_definition term_id="garbage-collection" length="short">}} 这
+允许像下面这样清理资源:
```
Comment (tengqm) [inline content/zh/docs/concepts/architecture/garbage-collection.md:9]: 翻译文档时，请参考其他文档的翻译方式和约定。
例如保留英文原文，用注释括起来。
链接使用相对 URL，不使用绝对地址。

**Pull request closed without merging**